docs/vagrant-spk/platform-stacks.md

```diff
@@ -6,6 +6,13 @@
 and/or web frameworks, each of which have different practices on how
 to go from the app's source code to a running web server.
 
+The following stacks exist:
+
+* `lemp`: a PHP-oriented software collection including nginx, MySQL, and PHP.
+* `meteor`: a stack for [Meteor](https://meteor.com) apps, including MongoDB.
+* `uwsgi`: a Python-oriented stack including nginx and uwsgi.
+* `diy`: Create your own.
```

Comment:
There's also `static`, which just installs nginx and serves the working directory, which is nice for testing entirely-javascript things that have no data and especially for testing `postMessage` shell integration.

But maybe that's not worth documenting.